OMCompiler/SimulationRuntime/c/simulation/solver/solver_main.h

```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of OpenModelica.
  *
- * Copyright (c) 1998-CurrentYear, Open Source Modelica Consortium (OSMC),
+ * Copyright (c) 1998-2020, Open Source Modelica Consortium (OSMC),
```

Comment:
From what I've read just now I think you only need to add the year of creation and don't  update it every year.
But I'm no lawyer, so I'll just revert the change.

Some guidelines for this would be great. Most stuff in C++ is missing the OSMC Public License Header anyway.